optional_plugins/varianter_yaml_to_mux/examples/tests/gendata.py

```diff
@@ -0,0 +1,18 @@
+import os
+
+from avocado import Test
+
+
+class GenDataTest(Test):
+
+    """
+    Simple test that generates data to be persisted after the test is run
+    """
+
+    def test_json(self):
+        import json
+        output_path = os.path.join(self.outputdir, "test.json")
+        output = {"basedir": self.basedir,
+                  "outputdir": self.outputdir}
+        with open(output_path, "w") as output_file:
+            json.dump(output, output_file)
```

Comment:
Hi @beraldoleal for this and the next 3 comments you made. These are files copied from `examples/tests` because the current tests from the plugins I migrated need them.